.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "1.0.0"
```

Comment:
Please change to `0.1.0`:
https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
Changed to empty json

Reply:
Great!